### ISSUE TITLE
Accessibility: Fix Elasticsearch config page fastpass issues

### ISSUE THREAD

**Issue by kaydelaney** [grafana/grafana#42708]

**Comment by Elfo404** [comment]
closing this ad it should be fixed with https://github.com/grafana/grafana/pull/43706
